benches/resolver.rs

```diff
@@ -1,35 +1,63 @@
 use criterion::{criterion_group, criterion_main, Criterion};
 use srv_rs::resolver::{libresolv::LibResolv, SrvResolver};
+use trust_dns_resolver::{AsyncResolver, TokioAsyncResolver};
 
 pub fn criterion_benchmark(c: &mut Criterion) {
     let mut runtime = tokio::runtime::Runtime::new().unwrap();
+    let libresolv = LibResolv::default();
+    let trust_dns = runtime
+        .block_on(AsyncResolver::tokio_from_system_conf())
+        .unwrap();
 
-    c.bench_function(
-        &format!("resolve {} (libresolv)", srv_rs::EXAMPLE_SRV),
-        |b| {
-            b.iter(|| {
-                runtime
-                    .block_on(LibResolv::default().get_srv_records_unordered(srv_rs::EXAMPLE_SRV))
-                    .unwrap()
-            })
-        },
-    );
+    let mut group = c.benchmark_group(format!("resolve {}", srv_rs::EXAMPLE_SRV));
+    group.bench_function("libresolv", |b| {
+        b.iter(|| {
+            runtime
+                .block_on(libresolv.get_srv_records_unordered(srv_rs::EXAMPLE_SRV))
+                .unwrap()
+        })
+    });
+    group.bench_function("trust-dns", |b| {
```

Comment:
Just fixed the benches so `trust-dns`' caching doesn't give it a huge advantage, and it now comes out ~4-6% slower than `libresolv`. Given that `trust-dns` is actually async, it'll probably perform better than `libresolv` in real applications.

Reply:
I should probably be spawning `libresolv` lookups on a threadpool or blocking threads in `tokio` to avoid the synchronous requests blocking async executors, but it'd be nice to avoid making anything runtime-specific

Reply:
ack